ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -309,6 +309,18 @@ export const itwEidIssuanceMachine = setup({
                   }
                 })),
                 target: "CieID"
+              },
+              {
+                guard: ({ event, context }) =>
+                  event.mode === "cieId" &&
+                  context.isL3FeaturesEnabled === true,
```

Comment:
Guards are evaluated in order, this guard should be placed before the `event.mode === "cieId"` otherwise it will never be tested.

https://stately.ai/docs/guards#multiple-guarded-transitions

Reply:
Thanks, fixed!